query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -31,16 +31,41 @@
 import org.codice.ddf.admin.configurator.OperationReport;
 import org.codice.ddf.internal.admin.configurator.actions.ManagedServiceActions;
 import org.codice.ddf.internal.admin.configurator.actions.ServiceActions;
+import org.codice.ddf.internal.admin.configurator.actions.ServiceReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ServiceCommons {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceCommons.class);
+
     public static final String SERVICE_PID_KEY = "service.pid";
 
     public static final String FACTORY_PID_KEY = "service.factoryPid";
 
-    // A flag to indicate if a service being updated has a password of "password". If so, the
+    // A flag to indicate if a service being updated has a password of "secret". If so, the
     // password will not be updated.
-    public static final String FLAG_PASSWORD = "password";
+    public static final String FLAG_PASSWORD = "secret";
+
+    private ManagedServiceActions managedServiceActions;
+
+    private ServiceActions serviceActions;
+
+    private ServiceReader serviceReader;
+
+    private ConfiguratorFactory configuratorFactory;
+
+    public ServiceCommons() {
```

Comment:
If we are going to pass in the service references for each method call, then we might as well keep all the methods static in this class.